packages/p2p/src/components/floating-rate/floating-rate.jsx

```diff
@@ -0,0 +1,115 @@
+import classNames from 'classnames';
+import { observer } from 'mobx-react-lite';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { InputField, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import './floating-rate.scss';
+
+const FloatingRate = ({
+    change_handler,
+    className,
+    exchange_rate,
+    error_messages,
+    fiat_currency,
+    local_currency,
+    offset,
+    placeholder,
+    ...props
+}) => {
+    const { general_store } = useStores();
+    const { name, value, required } = props;
+    const market_feed = value ? parseFloat(exchange_rate * (1 + value / 100)).toFixed(2) : exchange_rate;
+
+    return (
+        <div className={classNames(className, 'floating-rate')}>
+            <section className={classNames('floating-rate__field', { 'mobile-layout': isMobile() })}>
+                <InputField
+                    ariaLabel='Floating rate'
+                    classNameInlinePrefix='floating-rate__percent'
+                    classNameInput={classNames('floating-rate__input', {
+                        'floating-rate__input__error_field': error_messages,
+                    })}
+                    decimal_point_change={2}
```

Comment:
```suggestion
                    current_focus={ui.current_focus}
                    decimal_point_change={2}
```
Since you set current focus~ you should be using it